src/software/geom/util.cpp

```diff
@@ -265,28 +269,151 @@ bool intersects(const Segment &first, const Segment &second)
     return boost::geometry::intersects(AB, CD);
 }
 
-template <size_t N>
-Point getVertex(const LegacyPolygon<N> &poly, unsigned int i)
+std::vector<Shot> angleSweepCirclesAll(const Point &src, const Point &p1, const Point &p2,
+                                       const std::vector<Point> &obstacles,
+                                       const double &radius)
```

Comment:
Seems like github has the diff kinda funky here. Could you merge `master` into your branch? Hopefully then most of this bit of the diff should vanish. Otherwise, we might need to remove some stuff here, @EvanMorcom rewrote a lot of this.

Reply:
I rebased on master, Evan's changes are in the commit log for this branch. But if something is weird, let me know what I should remove

Reply:
I looked at Evan's changes and it seems like my merge reintroduced ```angleSweepCircles(...)``` and its helper. I removed these in the latest update.